crates/next-api/src/nft_json.rs

```diff
@@ -58,58 +71,62 @@ pub struct OutputSpecifier(Option<RcStr>);
 #[turbo_tasks::value_impl]
 impl NftJsonAsset {
     #[turbo_tasks::function]
-    async fn ident_in_project_fs(self: Vc<Self>) -> Result<Vc<FileSystemPath>> {
-        let this = self.await?;
-        let project_fs = this.project_fs.await?;
-        let output_fs = this.output_fs.await?;
-        let nft_folder = self.ident().path().parent().await?;
-
-        if let Some(subdir) = output_fs.root().strip_prefix(&**project_fs.root()) {
-            Ok(this
-                .project_fs
-                .root()
-                .join(subdir.into())
-                .join(nft_folder.path.clone()))
-        } else {
-            // TODO: what are the implications of this?
-            bail!("output fs not inside project fs");
-        }
+    async fn output_root_in_project_fs(self: Vc<Self>) -> Result<Vc<FileSystemPath>> {
+        Ok(self
+            .project_root()
+            .join(self.await?.project.project_path().await?.path.clone()))
     }
```

Comment:
This makes some assumptions about internals of the `Project`. Move this method into `Project`.